### DISCUSSION TITLE
Question about documentation.

### DISCUSSION THREAD

**Discussion by JaimePSantos** [JonathanVusich/pcpartpicker#54] [Q&A]
Hello! I am somewhat of a begginner in programming and i've been fiddling around with your API for a little project im working on. Is there any sort of documentation available?
Thank you very much!

**Comment by JonathanVusich** [comment]
I have not really put anything together yet. Examples for retrieving data can be found in the readme for this repository. The dataclasses that model the data scraped from PCPartPicker are not documented though. Right now, If you are using an IDE for your project you should be able to hop into the class definition and see all the different properties that they have available. If not, I can start working on generating some documentation for everything.

**Reply by JaimePSantos** [reply]
Thank you for the quick reply! 
After inspecting some of the files in the git i have managed to figure out how most of the stuff i need works. Again, great work, this tool is incredible!
I did, however, not manage to find the file that contains the Money class (moneyed i believe). I can always just convert the price to a string, remove the currency symbol and convert it to int (to play around with sorting by price and stuff), but i was wondering if the class had any methods for this, since it would be a more elegant solution.
Thanks!

--
Just figured out that the money class is included in the python instalation, nevermind.

**Reply by themodfather360** [reply]
> I have not really put anything together yet. Examples for retrieving data can be found in the readme for this repository. The dataclasses that model the data scraped from PCPartPicker are not documented though. Right now, If you are using an IDE for your project you should be able to hop into the class definition and see all the different properties that they have available. If not, I can start working on generating some documentation for everything.

Hi there, I'm not very familiar with python. All I really need is the correct command for accessing (or writing out) individual fields from the PartData stuff. Maybe it's obvious, but I don't know it. So far, I have:

cpu_data = api.retrieve("cpu")
print(cpu_data)

which prints a very long list. How can I access individual fields from the PartData (for example, cpu_data[2] for field 2?) Also, how to convert to json?
Thanks a lot.
